docs/UserGuide.md

```diff
@@ -92,7 +92,7 @@ Hope you'll be satisfied using NUSClasses :smile:
   e.g. in `addc n/CONTACTNAME`, `CONTACTNAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g. `addc n/CONTACTNAME p/PHONENUMBER e/EMAIL u/GIT_USERNAME [t/TAGS]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g. `addc n/CONTACTNAME p/PHONENUMBER e/EMAIL u/GIT_USERNAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```

Comment:
Do you think its appropriate to include the ellipses? Personally, feel its fine to include here since its in close proximity to the the usage of `ellipses`. Else, we could just swap it with the ellipses?
```suggestion
  e.g. `addc n/CONTACTNAME p/PHONENUMBER e/EMAIL u/GIT_USERNAME [t/TAG]...` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```